js/Makefile

```diff
@@ -356,14 +356,10 @@ ios/Frameworks/Bertybridge.framework: $(bridge_src)
 	cd .. && $(GO) mod download
 	cd .. && $(GO) install golang.org/x/mobile/cmd/gobind
 	cd .. && $(GO) run golang.org/x/mobile/cmd/gomobile init
-	# Download required go-libtor deps if needed
-	../tool/build/shell/dl-go-libtor-ios.sh
 	mkdir -p ios/Frameworks
```

Comment:
In this file you should also remove:
- `rm -rf ios/tor-deps` line 115
- `ios/OpenSSL-Universal-Override.podspec` deps of `ios/Berty.xcworkspace` rule on line 373

Reply:
Can you also remove the file `tool/build/shell/dl-go-libtor-ios.sh`?